src/components/FeatureInformation/FeatureInformation.scss

```diff
@@ -5,12 +5,12 @@
     justify-content: space-between;
     background-color: #f5f5f5;
     font-weight: bold;
-    height: 40px;
+    min-height: 40px;
 
     span {
       display: flex;
       align-items: center;
-      padding-left: 10px;
+      padding: 4px 0 4px 10px;
```

Comment:
I would prefer these core two changes as they adjust the title height for all topics, if necessary.